src/ReleaseHistory.md

```diff
@@ -11,6 +11,7 @@
 * BUGFIX: Update `merge` command to properly produce runs by tool and version when passed the `--merge-runs` argument. [#2488](https://github.com/microsoft/sarif-sdk/pull/2488)
 * BUGFIX: Eliminate `IOException` and `DirectoryNotFoundException` exceptions thrown by `merge` command when splitting by rule (due to invalid file characters in rule ids). [#2513](https://github.com/microsoft/sarif-sdk/pull/2513)
 * BUGFIX: Fix classes inside NotYetAutoGenerated folder missing `virtual` keyword for public methods and properties, by regenerate and manually sync the changes. [#2537](https://github.com/microsoft/sarif-sdk/pull/2537)
+* BUGFIX: MSBuild Converter accepts case insensitive keywords and supports PackageValidator msbuild log. [#2579](https://github.com/microsoft/sarif-sdk/pull/2579)
```

Comment:
>MSBuild Converter accepts case insensitive keywords and supports PackageValidator msbuild log [](http://example.com/codeflow?start=10&length=93)

It's super close! Use this:

MSBuild Converter now accepts case insensitive keywords and supports PackageValidator msbuild log output.

Reply:
thanks for the suggestion. updated.